Dockerfile

```diff
@@ -243,7 +243,7 @@ RUN apt-get --no-install-recommends install -y uidmap sudo vim iptables fuse \
   && update-alternatives --set iptables /usr/sbin/iptables-legacy
 # musl is needed to directly use the registry binary that is built on alpine
 ENV BUILDKIT_INTEGRATION_CONTAINERD_EXTRA="containerd-1.3=/opt/containerd-alt/bin"
-ENV BUILDKIT_INTEGRATION_CONTAINERD_STARGZ=1
+ENV BUILDKIT_INTEGRATION_STARGZ_SNAPSHOTTER=1
```

Comment:
`BUILDKIT_INTEGRATION_SNAPSHOTTER=stargz` ?

Reply:
Should we support multiple snapshotters? (e.g. `BUILDKIT_INTEGRATION_SNAPSHOTTER="stargz snapshotter-b snapshotter-c"`)

Reply:
Potentially in future

Reply:
Updated the test to make the snapshotter configurable through `BUILDKIT_INTEGRATION_SNAPSHOTTER`. For keeping the scope of this PR narrower, the test currently supports specifying single snapshotter through that envvar. But it shouldn't be difficult to extend this for multiple snapshotters if needed in the future.